### ISSUE TITLE
Add conversion to ISO

### ISSUE THREAD

**Issue by michelle1574** [corpnewt/gibMacOS#240]
Hello, @corpnewt.
I have a problem because I want to convert downloaded files into ISO, but media creator allows me only USB drive. Somebody can help me?

**Comment by KeygenLLC** [comment]
This works using Terminal on mac:

https://www.geekrar.com/create-macos-catalina-iso-file/

**Comment by michelle1574** [comment]
I MEAN ON WINDOWS!!! ADD CONVERSION TO ISO ON WINDOWS FROM PKG FILES!!! 😡😡😡😡😡😡

**Comment by Masamune3210** [comment]
Yelling will get you less than nowhere....

**Comment by Medizinmann-MD** [comment]
You could use qemu-convert. You can get qemu here: https://qemu.weilnetz.de/w64/

Fetch the BaseSystem.dmg and convert it o an .iso

Or you use dmg2img found here:
http://vu1tur.eu.org/tools/

How to use it found here:
https://www.tweaking4all.com/cd-dvd-blu-ray-disc-burning/burning-for-macosx/convert-dmg-to-iso/

**Comment by GF-Huang** [comment]
I'm curious why people always assume that the user of gibMacOS has a real mac, and if I have a real mac, what else do I download the dmg for.

**Comment by tianyehuakai** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by Anonym0us33** [comment]
Hi, farbeit for me to restart a flame war, but is there any solution to the initial issue?
> Add conversion to ISO #240

**Comment by Masamune3210** [comment]
> I'm curious why people always assume that the user of gibMacOS has a real mac, and if I have a real mac, what else do I download the dmg for.

They assume this because the only legitimate way to get the dmg is to have a real mac

**Comment by Medizinmann-MD** [comment]
> Hi, farbeit for me to restart a flame war, but is there any solution to the initial issue?
> 
> > Add conversion to ISO #240

Yes - I answered the question on Feb 2nd 2022 (https://github.com/corpnewt/gibMacOS/issues/240#issuecomment-1027802414).

But I would recommend to use some other method/tool than GibMacOS(this was okay up to Catalina though) to get an ISO like i.e. the guide from Nicholas Sherlock:
https://www.nicksherlock.com/2022/10/installing-macos-13-ventura-on-proxmox/
On Windows you would need to install WSL2 to use his method.

You could also go with the method from the OpenCore Install guide:
https://dortania.github.io/OpenCore-Install-Guide/installer-guide/windows-install.html#downloading-macos

Both method generate a recovery image - to get a full install image you would need a running MacOS.
